edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter/upstream/BrokeredCloudConnection.cs

```diff
@@ -4,26 +4,58 @@ namespace Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter
     using System;
     using System.Threading.Tasks;
     using Microsoft.Azure.Devices.Edge.Hub.Core.Cloud;
+    using Microsoft.Azure.Devices.Edge.Hub.Core.Identity;
     using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
 
     public class BrokeredCloudConnection : ICloudConnection
     {
+        IIdentity identity;
+
         public BrokeredCloudConnection(BrokeredCloudProxy cloudProxy)
         {
+            Preconditions.CheckNotNull(cloudProxy);
+
             this.IsActive = true;
             this.CloudProxy = Option.Some(cloudProxy as ICloudProxy);
+            this.identity = cloudProxy.Identity;
```

Comment:
This seems odd... can we just pass in the Identity separately, like we do for CloudConnection?